tests/test_Tag.cc

```diff
@@ -127,10 +128,15 @@ TEST_CASE("operator<<(std::ostream&, Tag)", "[Tag]")
     REQUIRE(stream.str() == "bloody sunday");
 }
 
-TEST_CASE("Tag: string()", "[Tag]")
```

Comment:
Why do you remove this test case?

Reply:
Oh. I have no idea how that happened, but it was not intentional. Thanks a lot!
Apparently I did not even check my own diff... well, in any case, it is fixed now.